frontend/src/views/containers/startGame/useGame.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from "react";
+import { ajax } from "rxjs/ajax";
+import { map } from "rxjs/operators";
+// eslint-disable-next-line no-restricted-imports
+import { Game } from "../../../domain/Game";
+//const ViewModel = require("../../model/ViewModel");
+
+function useGame() {
```

Comment:
čia viskas blogai apart to, kad panaudoji state ir useEffect'e subscribini, bet net value į state negerai įsidedi